versions.tf

```diff
@@ -0,0 +1,11 @@
+# Terraform version and plugin versions
+
+terraform {
+  required_version = "~> 0.11.10"
```

Comment:
How is this related to kubernetes version update?

Reply:
All the changes are imported from upstream. https://github.com/poseidon/terraform-render-bootkube//commit/0103bc06bb3f597455a765bf5d916f9b241cbea0